### PR TITLE
first implementation of the Bank Investment Contract (BIC) also named…

### PR DESCRIPTION
**Pull request by mothsART**
Inspiration here : https://github.com/Marak/faker.js/blob/46ad73d378697e06e03d73e38ed24ee9dcc97e8b/lib/finance.js#L300

You can try result on website like https://bank.codes/swift-code-checker/

### CODE REVIEW

File: fake/src/faker/impls/finance.rs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    "TV", "TW", "TZ", "UA", "UG", "UM", "US", "UY", "UZ", "VA", "VC",
+    "VE", "VG", "VI", "VN", "VU", "WF", "WS", "YE", "YT", "YU", "ZA",
+    "ZM", "ZR", "ZW"
+];
+
+const VOWELS: &'static [&'static str] = &["A", "E", "I", "O", "U"];
+
+impl<L: Data> Dummy<Bic<L>> for String {
+    fn dummy_with_rng<R: Rng + ?Sized>(_: &Bic<L>, rng: &mut R) -> Self {
+        let prob = (0..100).fake::<u32>();
```
Comment (cksac) [inline fake/src/faker/impls/finance.rs:44]: this should use `(0..100).fake_with_rng(rng)`  in order to have consistent result when passed rng

File: fake/src/faker/impls/finance.rs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                *ALPHABET.choose(rng).unwrap(),
+                *VOWELS.choose(rng).unwrap(),
+                *ALPHABET.choose(rng).unwrap()
+            );
+        } else if prob < 40 {
+            sufix = format!(
+                "{}{}{}",
+                (0..9).fake::<u32>().to_string(),
+                (0..9).fake::<u32>().to_string(),
+                (0..9).fake::<u32>().to_string()
```
Comment (cksac) [inline fake/src/faker/impls/finance.rs:58]: same as above, use `fake_with_rng`

File: fake/src/faker/impls/finance.rs (hunk 1 above)
Comment (mothsART) [reply fake/src/faker/impls/finance.rs]: correct : done

File: fake/src/faker/impls/finance.rs (hunk 2 above)
Comment (mothsART) [reply fake/src/faker/impls/finance.rs]: also done

**Pull request merged**